docs/spfx/web-parts/guidance/connect-to-sharepoint-using-jsom.md

```diff
@@ -375,14 +375,23 @@ Having loaded the titles of SharePoint lists in the current site, the final step
 
   ![SharePoint Framework client-side web part showing titles of SharePoint lists in the current site](../../../images/tutorial-spjsom-web-part-list-titles.png)
 
-Referencing SharePoint JSOM scripts declaratively as external scripts is convenient and allows you to keep your code clean. One disadvantage, however, is that it requires specifying absolute URLs to the location from which SharePoint JSOM scripts should be loaded. If you're using separate SharePoint tenants for development, testing, and production, it requires some additional work to change these URLs for the different environments accordingly. In such cases, you may consider referencing JSOM imperatively by using the [SPComponentLoader](https://docs.microsoft.com/javascript/api/sp-application-base) to load the scripts in the SPFx component's code.
+Referencing SharePoint JSOM scripts declaratively as external scripts is convenient and allows you to keep your code clean. One disadvantage, however, is that it requires specifying absolute URLs to the location from which SharePoint JSOM scripts should be loaded. If you're using separate SharePoint tenants for development, testing, and production, it requires some additional work to change these URLs for the different environments accordingly. In such cases, you may consider referencing JSOM imperatively by using the [SPComponentLoader](https://docs.microsoft.com/en-us/javascript/api/sp-loader/spcomponentloader) to load the scripts in the SPFx component's code.

```

Comment:
Can you remove the locale from the URL (requirements for MS Docs)... let the service route to the user's locale for localization.

Reply:
Sure! My bad... 
Done.